internal/validations/clusterlogging/validate_clusterlogging_spec.go

```diff
@@ -15,8 +21,16 @@ func validateClusterLoggingSpec(cl v1.ClusterLogging) error {
 		return nil
 	}
 	spec := cl.Spec
+
 	if spec.Forwarder != nil || spec.LogStore != nil || spec.Curation != nil || spec.Visualization != nil {
-		return errors.NewValidationError("Only spec.collection is allowed when using multiple instances of ClusterLogForwarder: %s/%s", cl.Namespace, cl.Name)
+		key := types.NamespacedName{Name: cl.Name, Namespace: cl.Namespace}
+		clf := runtime.NewClusterLogForwarder(cl.Namespace, cl.Name)
+		if err := k8sClient.Get(context.TODO(), key, clf); err != nil {
+			if apierrors.IsNotFound(err) {
+				return errors.NewValidationError("ClusterLogForwarder: %s/%s instance not found, ClusterLogging object in %q not named %q also "+
```

Comment:
This should change based upon my first comment.  As-is the message is really long and difficult to understand